install/install.md

```diff
@@ -239,12 +263,12 @@ bash /rapids/utils/stop-jupyter.sh
 <div id="pip"></div>
 
 ## **pip**
-RAPIDS 23.06 pip packages of cuDF, dask-cuDF, cuML, cuGraph, cuSpatial, RMM, and RAFT are available for CUDA 11 and CUDA 12 on the NVIDIA Python Package Index.
+RAPIDS pip packages are available for CUDA 11 and CUDA 12 on the NVIDIA Python Package Index.
 
 ### **pip Additional Prerequisites**
 <i class="fas fa-info-circle"></i> The CUDA toolkit version on your system must match the pip CUDA version you install (`-cu11` or `-cu12`). <br/>
 <i class="fas fa-info-circle"></i> **glibc version:** x86_64 wheels require glibc >= 2.17. <br/>
-<i class="fas fa-info-circle"></i> **glibc version:** ARM architecture (aarch64) wheels require glibc >= 2.31 (only ARM Server Base System Architecture is supported).
+<i class="fas fa-info-circle"></i> **glibc version:** ARM architecture (aarch64) wheels require glibc >= 2.32 (only ARM Server Base System Architecture is supported).
```

Comment:
Is this true for CUDA 11 or only CUDA 12? (Or is this the glibc requirement from dask-sql?)

I'm surprised because Ubuntu 20.04 wouldn't even meet this criterion, because [Ubuntu 20.04 uses glibc 2.31](https://packages.ubuntu.com/focal/glibc-source). That'd be very confusing, because I think ARM + Ubuntu 20.04 works fine in practice... (I haven't tested this and would like clarification if anyone knows).

Reply:
I did a bit more research and found that CUDA 12.0.1 supports glibc >=2.28 on ARM. https://docs.nvidia.com/cuda/archive/12.0.1/cuda-installation-guide-linux/index.html

This is my understanding from the conda-forge side of things too, I think we just need glibc >=2.28 for CUDA 12.

Reply:
We actually didn't make Ubuntu 20.04 images for ARM in 23.06: https://github.com/rapidsai/docker/pull/551

The issue came up during notebook testing during release, maybe overlooked for a little while but with the overhaul notebook testing gets run as part of the build so hopefully shouldn't happen again.

Reply:
Okay, thanks for the clarification. That sounds right, it’s not just a CUDA requirement but also that bug.

Reply:
(I mentioned dask-sql above, it’s the same bug here.)

Reply:
The Dask-SQL GLIBC bug was recently fixed: https://github.com/dask-contrib/dask-sql/pull/1201